.github/workflows/test-admin-interface.yml

```diff
@@ -0,0 +1,95 @@
+name: Gradido Admin Interface Test CI
+
+on: push
+
+jobs:
+  # only (but most important) job from this workflow required for pull requests
+  # check results serve as run conditions for all other jobs here 
+  files-changed:
+    name: Detect File Changes - Admin Interface
+    runs-on: ubuntu-latest
+    outputs:
+      admin: ${{ steps.changes.outputs.admin }}
+    steps:
+      - uses: actions/checkout@v3.3.0
+
+      - name: Check for admin interface file changes
+        uses: dorny/paths-filter@v2.11.1
+        id: changes
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+          list-files: shell
+
+  
+  build_test:
+    if: needs.files-changed.outputs.admin == 'true'
+    name: Docker Build Test - Admin Interface
+    needs: files-changed
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Admin Interface | Build 'test' image
+        run: docker build --target test -t "gradido/admin:test" admin/ --build-arg NODE_ENV="test"
+
+  unit_test:
+    if: needs.files-changed.outputs.admin == 'true'
+    name: Unit Tests - Admin Interface
+    needs: files-changed
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Admin Interface | Unit tests
+        run: |
+          cd admin && yarn && yarn run test
+          cp -r ./coverage ../
+
+      - name: Admin Interface | Coverage check
+        uses: webcraftmedia/coverage-check-action@master
+        with:
+          report_name: Coverage - Admin Interface
+          type: lcov
+          result_path: ./admin/coverage/lcov.info
+          min_coverage: 97
+          token: ${{ github.token }}
```

Comment:
```suggestion
          token: ${{ github.token }}
          
    coverage_skip:
      if: needs.files-changed.outputs.admin == 'false'
      push job with name `Coverage - Admin Interface` as skipped
```

Reply:
The coverage check dependencies is resolved since #2790 has been merged.